src/UserService.Validation/User/Certificates/CreateCertificateRequestValidator.cs

```diff
@@ -0,0 +1,33 @@
+﻿using FluentValidation;
+using LT.DigitalOffice.UserService.Data.Interfaces;
+using LT.DigitalOffice.UserService.Models.Dto.Requests.User.Certificates;
+
+namespace LT.DigitalOffice.UserService.Validation.User.Certificates
+{
+    public class CreateCertificateRequestValidator : AbstractValidator<CreateCertificateRequest>
+    {
+        public CreateCertificateRequestValidator(IUserRepository repository)
+        {
+            RuleFor(x => x.Name)
+                .NotEmpty()
```

Comment:
Do you write an error message for empty value?

Reply:
There are default message